### ISSUE TITLE
ppa for Ubuntu 22.04. (jammy)

### ISSUE THREAD

**Issue by rcassani** [blmhemu/opengigabyte#15]
### Discussed in https://github.com/blmhemu/opengigabyte/discussions/12

<div type='discussions-op-text'>

<sup>Originally posted by **Arny80Hexa** September 20, 2023</sup>
After two years with my Aorus 15P, I stumbled across your repository today and am super happy that I finally got the screen brightness keys (Fn+F3/F4) to work. A big thank you for your work!

Unfortunately there is currently no ppa for my distribution and so I was forced to compile your drivers by hand into my kernel. I use the current Linux Mint version which uses Ubuntu 22.04. (jammy) as a base.

And so I would like to ask you: would it be possible for you to update your ppa to Ubuntu 22.04.?</div>

**Comment by Micheallscarn** [comment]
@rcassani  hey! 
just came across your repos for Aorus 15P and finally, i have some hope to get them to work :) 
btw , Do you know how can i install this repo for fedora 37 ? 
and did the fan control repo that you did , does it have a UI, or do we have to do the process each time? 
thank you so much for doing this!

**Comment by SIMULATAN** [comment]
Hi @Micheallscarn 

I actually also have the Aorus 15P (KD)! I unfortunately can't help you with the Fedora installation, however, I do have some information regarding the fan control.
His project only features a very low-level interface to the controller but he thankfully implemented the hardest part of all: reverse-engineering the registers, data types, lengths etc.
If you want a high-level tool, checkout [my AorusControl project](https://github.com/SIMULATAN/AorusControl). It currently allows you to monitor your fans in a TUI and select a mode, but I may add more in the future (including a GUI, if someone is interested..?). Please try it out and let me know if you have any improvement ideas!
![screenshot of AorusControl](https://github.com/blmhemu/opengigabyte/assets/63104422/c27800fc-8c9d-4ada-9eef-096a360d6379)

**Comment by Micheallscarn** [comment]
@SIMULATAN oh hi! 
thanks for the replay , I will try it , although I don't see an README file so can you kindly walk me through it

**Comment by SIMULATAN** [comment]
> @SIMULATAN oh hi! thanks for the replay , I will try it , although I don't see an README file so can you kindly walk me through it

Yep, sorry, I didn't make one till just now. Now, I pushed it + [made a release](https://github.com/SIMULATAN/AorusControl/releases/tag/1.0.0). See the new `Installation` and `Usage` instructions in [the README](https://github.com/SIMULATAN/AorusControl#installation). Let me know if anything's unclear!

Also, feel free to open an issue there if you have any problems to avoid spamming this one with unrelated off-topic talk and to help users of my project.
